### PR TITLE
adding tag-cloud components

### PR DESCRIPTION
**Pull request by HunkevychPhilip**
An initial set of tag-cloud component files

### CODE REVIEW

File: src/Controller/TagCloudController.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+class TagCloudController extends AbstractController
+{
+    /**
+     * @Route("/tagcloud", name="tag_cloud")
+     */
+    public function index()
+    {
+        $data = $this->getData();
+        return $this->render('tag_cloud/index.html.twig', [
```
Comment (yderevianko-cartera) [inline src/Controller/TagCloudController.php:18]: missed empty line before return

File: src/Controller/TagCloudController.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            'controller_name' => 'TagCloudController', 'data' => $data
+        ]);
+    }
+    public function getData()
```
Comment (yderevianko-cartera) [inline src/Controller/TagCloudController.php:24]: should be empty line before new function/method

File: src/Controller/TagCloudController.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        $maximum = 0;
+
+        foreach ($searches as $search) {
+
+            $counter = $search['searched'];
+
+            // dd($search);
+            $class = '';
+
+            if ($counter > $maximum) $maximum = $counter;
```
Comment (yderevianko-cartera) [inline src/Controller/TagCloudController.php]: always use { and } in if statements and remove not needed empty lines and comments

File: src/Repository/SearchesRepository.php [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    {
+        parent::__construct($registry, Searches::class);
+    }
+
+    // /**
+    //  * @return Searches[] Returns an array of Searches objects
+    //  */
+    public function findTopHighestSearch()
+    {
+
```
Comment (yderevianko-cartera) [inline src/Repository/SearchesRepository.php]: not needed empty lines

**Review by yderevianko-cartera — CHANGES_REQUESTED** [review]
Please fix merge conflicts

**Review by CoremanY — APPROVED** [review]

**Pull request merged**